.github/workflows/test_all_browsers.yml

```diff
@@ -13,6 +13,11 @@ on:
         description: 'Skip all tests'
         default: false
         type: boolean
+      max-attempts:
+        description: 'Max attempts to retry test when test failed'
+        required: false
+        default: '0'
+        type: string
```

Comment:
Why .github/workflows/test_all_browsers.yml has type but .github/actions/run-test/action.yml doesn't have type?

Reply:
GitHub does not provide the `type` API  for `Composite Action`, which differs from `Reuseable Workflow`.